app/views/govuk_publishing_components/components/_metadata.html.erb

```diff
@@ -11,9 +11,11 @@
   direction_class = ""
   direction_class = " direction-#{direction}" if local_assigns.include?(:direction)
 
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
   classes = %w(gem-c-metadata)
   classes << "direction-#{direction}" if local_assigns.include?(:direction)
   classes << "gem-c-metadata--inverse" if inverse
+  classes << shared_helper.get_margin_bottom if local_assigns[:margin_bottom]
```

Comment:
Rather than the component setting its own margin values in component CSS, it seems like the values are defined in [this mixin](https://github.com/alphagov/govuk_publishing_components/blob/5c2c81a85d42de15664fe50a54a94e3db73ea0ae/app/assets/stylesheets/govuk_publishing_components/components/mixins/_margins.scss#L1)  which seems to be used in a few places in the gem. 

I am not quite sure why the mixin uses non-standard spacing values though, I wonder if it pre-dates the spacing scale perhaps?